Adding Diagrams into the docs for as source (#2314)
dapr/docs: daprdocs/content/en/contributing/contributing-docs.md

- **Assume a new developer audience** - Some obvious steps can seem hard. E.g. Now set an environment variable Dapr to a value X. It is better to give the reader the explicit command to do this, rather than having them figure this out.
- **Use present tense** - Avoid sentences like "this command will install redis", which implies the action is in the future. Instead use "This command installs redis" which is in the present tense.

## Diagrams and images

> It is strongly encouraged to create diagrams and images where ever possible for documentation pages. All diagrams are kept in a [Dapr Diagram Deck](images/dapr-diagrams.pptx) which has guidance on style and icons. The diagram images are saved as PNG files into the [images folder](/images). 

Author: ```suggestion
It is strongly encouraged to create diagrams and images where ever possible for documentation pages. All diagrams are kept in a [Dapr Diagram Deck](/images/dapr-diagrams.pptx) which has guidance on style and icons. The diagram images are saved as PNG files into the [images folder](/images). 
```

> It is strongly encouraged to create diagrams and images whereever possible for documentation pages. All diagrams are kept in a [diagram Powerpoint Deck](https://github.com/dapr/docs/tree/v1.6/daprdocs/static/images/dapr-diagrams.pptx) which has guidance on style and icons. The diagram images are saved as PNG files into the [images folder](https://github.com/dapr/docs/tree/v1.6/daprdocs/static/images). 

Author: ```suggestion
It is strongly encouraged to create diagrams and images where ever possible for documentation pages. All diagrams are kept in a [diagram Powerpoint Deck](https://github.com/dapr/docs/tree/v1.6/daprdocs/static/images/dapr-diagrams.pptx) which has guidance on style and icons. The diagram images are saved as PNG files into the [images folder](https://github.com/dapr/docs/tree/v1.6/daprdocs/static/images). 
```

> It is strongly encouraged to create diagrams and images where ever possible for documentation pages. All diagrams are kept in a [diagram Powerpoint Deck](https://github.com/dapr/docs/tree/v1.6/daprdocs/static/images/dapr-diagrams.pptx) which has guidance on style and icons. The diagram images are saved as PNG files into the [images folder](https://github.com/dapr/docs/tree/v1.6/daprdocs/static/images). 

Author: ```suggestion
It is strongly encouraged to create diagrams and images where ever possible for documentation pages. All diagrams are kept in a [Dapr Diagram Deck](images/dapr-diagrams.pptx) which has guidance on style and icons. The diagram images are saved as PNG files into the [images folder](/images). 
```

Diagrams should be;
- Saved as PNG files with a high resolution
- Named using the convention of a concept or building block so that they are grouped. For example `service-invocation-overview.png`. Also see Images guidance section below.
- Added to the correct section in the `Dapr-Diagrams.pptx` deck so that they can be amended and updated.

> 

Author: {{< button text="Download the Dapr Diagrams Deck" link="/images/dapr-diagrams.pptx" >}}

> 

Author: ```suggestion

{{< button text="Download the Dapr Diagrams Deck" link="/images/dapr-diagrams.pptx" >}}

```

- **Assume a new developer audience** - Some obvious steps can seem hard. E.g. Now set an environment variable Dapr to a value X. It is better to give the reader the explicit command to do this, rather than having them figure this out.
- **Use present tense** - Avoid sentences like "this command will install redis", which implies the action is in the future. Instead use "This command installs redis" which is in the present tense.

## Diagrams and images

> It is strongly encouraged to create diagrams and images where ever possible for documentation pages. All diagrams are kept in a [Dapr Diagram Deck](/images/dapr-diagrams.pptx) which has guidance on style and icons. The diagram images are saved as PNG files into the [images folder](/images). 

Author: ```suggestion
It is strongly encouraged to create diagrams and images where ever possible for documentation pages. All diagrams are kept in a [Dapr Diagrams Deck](/images/dapr-diagrams.pptx) which has guidance on style and icons. The diagram images are saved as PNG files into the [images folder](/images). 
```
